packages/in_app_purchase/in_app_purchase_android/pigeons/messages.dart

```diff
@@ -12,14 +12,25 @@ import 'package:pigeon/pigeon.dart';
   copyrightHeader: 'pigeons/copyright.txt',
 ))
 
-/// Pigeon version of Java Product.
-class PlatformProduct {
-  PlatformProduct({required this.productId, required this.productType});
+/// Pigeon version of Java QueryProductDetailsParams.Product.
+class PlatformQueryProduct {
```

Comment:
This rename is because I discovered that there are at least two different things called `Product` in the Billing Client API surface, so the old name was too generic.